ui/src/providers/Variants.js

```diff
@@ -185,61 +185,55 @@ export const useVariants = () => {
     const modelsSqon = { op: 'in', content: { fieldName: 'name', value: modelName } };
 
     const response = await apiFetcher({
-      body: { query, queryName: 'VariantsData', variables: { sqon: modelsSqon } },
+      body: { query, variables: { sqon: modelsSqon } },
       endpointTag: 'VariantsData',
     });
 
-    const data = get(response, `data.model.hits.edges[0].node.genomic_variants.hits.edges`, []).map(
-      ({ node }) => node,
-    );
+    const data = get(response, `data.model.hits.edges[0].node.variants.hits.edges`, [])
```

Comment:
Nice catch, I completely missed this `genomic_variants` here in the previous PRs!